### PR TITLE
Update Limit Reductions Table Form to display duration bounds in minutes and adjust translations accordingly

### PR DESCRIPTION
**Pull request by jamal-khey**

### CODE REVIEW

File: src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx [hunk 1]
```diff
@@ -21,11 +21,18 @@ const LimitReductionsTableForm: FunctionComponent<{
 
     const getLabelColumn = useCallback(
         (limit: ITemporaryLimitReduction) => {
+            const lowBound = Math.trunc(limit.limitDuration.lowBound / 60);
+            const highBound = Math.trunc(limit.limitDuration.highBound / 60);
+            if (lowBound === 0) {
+                return intl.formatMessage({ id: 'LimitDurationAfterIST' }, { value: highBound });
+            }
+            const lowBoundLabel = lowBound === 0 ? 'IST' : 'IT ' + lowBound;
```
Comment (SlimaneAmar) [inline src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx:29]: ?
IST it's for permanent limit

File: src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx [hunk 2]
```diff
@@ -21,11 +21,18 @@ const LimitReductionsTableForm: FunctionComponent<{
 
     const getLabelColumn = useCallback(
         (limit: ITemporaryLimitReduction) => {
+            const lowBound = Math.trunc(limit.limitDuration.lowBound / 60);
```
Comment (SlimaneAmar) [inline src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx:24]: Need tooltips ?

File: src/translations/fr.json [hunk 3]
```diff
@@ -1,9 +1,11 @@
 {
     "General": "General",
 
-    "LimitReductions" : "Abattements de limites",
+    "LimitReductions" : "Abattements",
```
Comment (SlimaneAmar) [inline src/translations/fr.json:4]: Rename by `Reduction` ?

File: src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx [hunk 4]
```diff
@@ -21,34 +21,47 @@ const LimitReductionsTableForm: FunctionComponent<{
...
-                    value: Math.trunc(limit.limitDuration.lowBound / 60),
+                    lowBound: `IT ${lowBound}`,
+                    highBound: `IT ${highBound}`,
                 }
             );
         },
         [intl]
     );
 
+    const getToolTipColumn = useCallback((limit: ITemporaryLimitReduction) => {
```
Comment (SlimaneAmar) [inline src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx:40]: Remove `useCallback` ?

File: src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx [hunk 5]
```diff
@@ -21,34 +21,47 @@ const LimitReductionsTableForm: FunctionComponent<{
...
+        const lowBound = Math.trunc(limit.limitDuration.lowBound / 60);
+        const highBound = Math.trunc(limit.limitDuration.highBound / 60);
+        return `[${lowBound} min, ${highBound} min[`;
```
Comment (SlimaneAmar) [inline src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx:43]: I think inclusion or exclusion ([ or ]) depends on lowClosed and highClosed

File: src/translations/en.json [hunk 6]
```diff
@@ -3,7 +3,9 @@
 
     "LimitReductions" : "Limit reductions",
     "IST" : "IST",
-    "LimitDuration" : "Limit duration {sign} {value}min",
+    "LimitDuration" : "Between {highBound} and {lowBound}",
```
Comment (SlimaneAmar) [inline src/translations/en.json:6]: `LimitDurationInterval` ?

File: src/translations/fr.json [hunk 7]
```diff
@@ -3,7 +3,8 @@
 
     "LimitReductions" : "Abattements de limites",
     "IST" : "IST",
-    "LimitDuration" : "Surcharge {sign} {value}min",
+    "LimitDuration" : "Entre {highBound} et {lowBound}",
+    "LimitDurationAfterIST": "Audelà de IT{value}",
```
[the remark this answers is not in the document]
Comment (jamal-khey) [reply src/translations/fr.json]: fixed

File: src/translations/fr.json (hunk 3 above)
Comment (jamal-khey) [reply src/translations/fr.json:4]: same as the previous remark i think its better to keep it as it is

File: src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx [hunk 8]
```diff
@@ -14,20 +14,36 @@ import {
...
-                { id: 'LimitDuration' },
+                { id: 'LimitDurationInterval' },
                 {
-                    sign: limit.limitDuration.lowClosed ? '>=' : '>',
-                    value: Math.trunc(limit.limitDuration.lowBound / 60),
+                    lowBound: lowBound === 0 ? 'IST' : `IT${lowBound}`,
+                    highBound: highBound === 0 ? 'Permanent' : `IT${highBound}`,
                 }
             );
+            // return getLabelColumn(limit);
```
Comment (jamal-khey) [inline src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx:46]: to remove

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**